### DISCUSSION TITLE
Bforartists 4 - Version 4.1.1

### DISCUSSION THREAD

**Discussion by Draise14** [Bforartists/Bforartists#4224] [Releases]
## Bforartists 4 version 4.1.1 is a hot-fix release

This release has a new feature to assign Collection Exporters for batch exports and another 24 task and features added to everything from the latest from Blender 4.1.1 and 4.2.0 Alpha to the date. 

The binary downloads are attached. Per usual they can also be found in the download section of the official website: 

### 📥 Download Free
https://www.bforartists.de/download/

### 📝 Release Notes
https://www.bforartists.de/release-notes/

### 🧰 Source Code
https://github.com/Bforartists/Bforartists

[Blender Release](https://www.blender.org/download/releases/) notes now merged and updated into Bforartists:

![Bforartists 4 1 0 Splash-large](https://github.com/Bforartists/Bforartists/assets/35357409/436fea9a-bc65-49bd-a45c-585d59a2c83b)

![Bforartists 4 1 1 Screenshot](https://github.com/Bforartists/Bforartists/assets/35357409/b2df83af-a3da-47e5-867a-76bb5e9cb3f1)

Splashscreen is kindly provided by Belzunze, the latest winner of the Splashscreen Competition.

Thanks again for all your art, feedback and support! And a very very very special thank you to the developers maintaining this.

_Draise, Reiner, Iyad, Blenux, Sean and the rest of the team_

**Full Changelog**: https://github.com/Bforartists/Bforartists/compare/v4.1.0...v4.1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/Bforartists/Bforartists/releases/tag/v4.1.1'>Bforartists 4 - Version 4.1.1</a>.</em>

**Comment by olivergamevy** [comment]
The mac silicon app seems to be corrupted and wont start.

**Comment by ReinerBforartists** [comment]
Many thanks for the report. Have you tried the tips that are mentioned at the download page?

> In case the silicon version does not start, make sure the App Store and identified developers option is ticked. And check if the files and folders tab to see Bforartists is listed and has permissions. Further informations can be found here: [support.apple.com/en-au/HT202491](https://support.apple.com/en-au/HT202491)
> In case it still does not work, try to launch from terminal with **xattr -cr**.

**Reply by olivergamevy** [reply]
Those steps don't work. I get the error '"Bforartists 4.1.1 mac silicon" is damaged and can't be opened. You should move it to the Bin."

**Reply by Draise14** [reply]
I'll see if one of the volunteers can make another build.

**Reply by rbq** [reply]
Just discovered this project, but couldn't find a Homebrew cask and the manual download is still broken. 😔

**Reply by Draise14** [reply]
@rbq  The manual is quite large (114 MB), so maybe downloading the individual chapters may help. You can find them here:
https://www.bforartists.de/bforartists-2-reference-manual/

About a Homebrew cask, is this a package manager for MacOS? At the moment we haven't supported that, to be honest first time I've heard of it! But drop by the discord server and you can talk directly with our Mac maintainers, we all volunteer the maintenance for Mac there. 
**💜  Discord**
https://discord.gg/yKuR77v

**Reply by rbq** [reply]
@Draise14 Oops, sorry, what I meant was: Downloading and installing the executable for Apple M CPUs manually doesn't work. 😅
